sdk/resourcemanager/azure-resourcemanager-storage/src/main/java/com/azure/resourcemanager/storage/implementation/StorageAccountsImpl.java

```diff
@@ -40,7 +41,11 @@ public CheckNameAvailabilityResult checkNameAvailability(String name) {
 
     @Override
     public StorageAccountImpl define(String name) {
-        return wrapModel(name).withSku(StorageAccountSkuType.STANDARD_RAGRS).withGeneralPurposeAccountKindV2();
+        return wrapModel(name)
+            .withSku(StorageAccountSkuType.STANDARD_RAGRS)
+            .withGeneralPurposeAccountKindV2()
+            .withOnlyHttpsTraffic()
+            .withMinimumTlsVersion(MinimumTlsVersion.TLS1_2);
```

Comment:
user still able to customize this one, right?

Reply:
Yes, customer can configure differently as in test code https://github.com/Azure/azure-sdk-for-java/pull/19649/files#diff-edf813069045825a9ed68c4408e389ba75eae39461b4a2ff37b8a5df20469e05R246-R250